README.md

```diff
@@ -5,6 +5,20 @@
 [![Coverage](https://coveralls.io/repos/github/TuringLang/AdvancedPS.jl/badge.svg?branch=master)](https://coveralls.io/github/TuringLang/AdvancedPS.jl?branch=master)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 
+AdvancedPS provides and efficient implementation of common particle based Monte Carlo samplers using the [AbstractMCMC](https://github.com/TuringLang/AbstractMCMC.jl) interface.
+AdvancedPS is part of the [Turing](https://turing.ml/stable/) ecosystem.
+
+### Installation
+
+Inside the julia REPL
```

Comment:
IIRC the convention is to use uppercase for the language and lowercase only for the binary:
```suggestion
Inside the Julia REPL
```